core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -156,47 +165,96 @@ object ConsumerGroupCommand {
       val props = if (opts.options.has(opts.commandConfigOpt)) Utils.loadProps(opts.options.valueOf(opts.commandConfigOpt)) else new Properties()
       val channelSocketTimeoutMs = props.getProperty("channelSocketTimeoutMs", "600").toInt
       val channelRetryBackoffMs = props.getProperty("channelRetryBackoffMsOpt", "300").toInt
+      if (!zkUtils.getConsumerGroups().contains(group)) {
+        println("The consumer group provided does not exist")
```

Comment:
That's a good suggestion. Will update this one and the one a few lines below to return the group name as part of the output message.